gdal/data/gdalvrt.xsd

```diff
@@ -210,6 +210,8 @@
         </xs:sequence>
         <xs:attribute name="dataType" type="DataTypeType"/>
         <xs:attribute name="band" type="xs:unsignedInt"/>
+        <xs:attribute name="blockXSize" type="xs:unsignedInt"/>
+        <xs:attribute name="blockYSize" type="xs:unsignedInt"/>
         <xs:attribute name="subClass" type="VRTRasterBandSubClassType"/>
 
         <xs:attribute name="BlockXSize" type="xs:unsignedInt"/> <!-- ignored -->
```

Comment:
is it intentional to add blockXSize when BlockXSize is already present?

Reply:
ah, good catch. I guess I put them there as a way of reserving them for the future. I've just removed that reservation, and adopt the syntax with leading letter in underscore for consistency with other attributes